doc/sphinx/espressopp.io.DumpH5MDParallel.rst

```diff
@@ -0,0 +1,38 @@
+Parallel H5MD Output
+====================
+
+H5MD_ is a format specification on top of the HDF5_ file format.
+
+.. _H5MD: https://nongnu.org/h5md/
+.. _HDF5: https://www.hdfgroup.org/solutions/hdf5/
+
+Dump Simulation
+---------------
+
+Dumping a simulation with ``espressopp.io.DumpH5MDParallel``
+
+::
+
+    dump_h5md_parallel = espressopp.io.DumpH5MDParallel(system, 'dump.h5')
+    dump_h5md_parallel.dump()
+
+
```

Comment:
Let's add here 
1. the possible options on what can be stored and how to enable/disable it.
2. the path to the particles location as it is fixed (`/particles/atoms/`)

I would also add *Attention* or *Note* with information that the dump is valid only for V=const (NVT, NVE) simulation as we do not save box dimensions at every simulation step.

Reply:
done